msteams-platform/concepts/device-capabilities/media-capabilities.md

```diff
@@ -137,6 +137,77 @@ microsoftTeams.media.selectMedia(mediaInput, (error: microsoftTeams.SdkError, at
 });
 ```
 
+* Call `selectMedia` API for capturing videos using camera:
+
+  * Capturing videos with `fullscreen: true`:
```

Comment:
`fullscreen: true` looks a bit confusing. The parameter name is different and this also looks like a drama name only. Can we  use something like : Capture videos in full Screen ? Just a suggestion, Mukul can confirm